km_api/know_me/models.py

```diff
@@ -355,6 +393,111 @@ def has_object_write_permission(self, request):
         return request.user == self.user
 
 
+class EmergencyContact(mixins.IsAuthenticatedMixin, models.Model):
+    """
+    An emergency contact holds contact information for emercency situations
+
+    Attributes:
+        km_user:
+            Emergency Contacts parent ``KMUser``.
+        name (str):
+            The emergency contacts's name.
+        relation (str):
+            Emergency contats relationship with the profile owner.
+        phone_number (big int):
+            The emergency contacts phone number.
+        alt_phone_number (optional):
+            Second optional phone number for the emergency contact.
+        email (optional):
+            The emergency contact's email address.
+    """
+    km_user = models.ForeignKey(
+        'know_me.KMUser',
+        related_name='emergency_contacts',
+        related_query_name='emergency_contact',
+        verbose_name=_('know me user'))
+    name = models.CharField(
+        max_length=255,
+        verbose_name=_('name'))
+    relation = models.TextField(
+        max_length=255,
+        verbose_name=_('relation'))
+    phone_number = models.CharField(
+        validators=[
+            RegexValidator(regex='^1?\d{9,15}$',
+                           message='Enter valid phone number',
+                           code='invaid_phone_number'), ],
+        max_length=15,
+        verbose_name=_('phone number'))
+    alt_phone_number = models.CharField(
+        validators=[
+            RegexValidator(regex='^1?\d{9,15}$',
+                           message='Enter valid phone number',
+                           code='invaid_phone_number'), ],
+        max_length=15,
+        blank=True,
+        verbose_name=_('alternate phone number'))
+    email = models.EmailField(
+        blank=True,
+        default='',
+        verbose_name=_('email'))
+
+    class Meta:
+        verbose_name = _('emergency contact')
+        verbose_name_plural = _('emergency contact')
+
+    def __str__(self):
+        """
+        Get a string representation of the emergency contact.
+
+        Returns:
+            str:
+                The emergency contact's name.
+        """
+        return self.name
+
+    def get_absolute_url(self):
+        """
+        Get the URL of the profile item's detail view.
+
+        Returns:
+            str:
+                The absolute URL of the profile item's detail view.
+        """
+        return reverse('know-me:emergency-contact-detail',
+                       kwargs={'pk': self.pk})
```

Comment:
Stylistically, we have been writing function calls with multi-line arguments as:
```python
reverse(
    'know-me:emergency-contact-detail',
    kwargs={'pk': self.pk})
```